services/apps/alcs/src/portal/application-submission-review/application-submission-review.controller.ts

```diff
@@ -368,23 +370,10 @@ export class ApplicationSubmissionReviewController {
 
       await this.applicationSubmissionReviewService.delete(applicationReview);
 
-      await this.applicationSubmissionStatusService.setStatusDate(
-        applicationSubmission.uuid,
-        SUBMISSION_STATUS.RETURNED_TO_LG,
-        null,
-      );
-
-      await this.applicationSubmissionStatusService.setStatusDate(
-        applicationSubmission.uuid,
-        SUBMISSION_STATUS.SUBMITTED_TO_LG,
-        null,
-      );
-
-      await this.applicationSubmissionStatusService.setStatusDate(
-        applicationSubmission.uuid,
-        SUBMISSION_STATUS.IN_REVIEW_BY_LG,
-        null,
-      );
+      //Clear Submission Date
```

Comment:
So you are not cleaning the dateSubmittedToAlcs status date?

Reply:
Good point! I should be clearing the submitted to ALC date when the ALC returns to L/FNG.

Reply:
Updated to make more sense, we now clear the submitted date at the same time/place where we clear the status.